### ISSUE TITLE
Can't open files with Collabora Online Development Edition

### ISSUE THREAD

**Issue by JackFish1** [nextcloud/android#4241]
### Actual behaviour
- Infinitive spinning circle when trying to open e.g. .odt-files within Android app

### Expected behaviour
- file opened and ready for editing, as it does without problems via web-access
 
### Steps to reproduce
1. goto file 
2. press "Öffnen mit Collabora Online Development Edition"
3. spinning circle

### Environment data
Android version: 6.0.1

Device model: Xperia Z3 Compact

Stock or customized system:
Apache2: 2.4.29 with enabled capabilities in config :
{"convert-to":{"available":false},"hasMobileSupport":true,"hasTemplateSaveAs":true,"productName":"Collabora Online Development Edition"}

LOOLWSD: 4.0.4

I do not find any Logfiles in App-Version 3.7.0.

Nextcloud app version: 3.7.0

Nextcloud server version: 16.0.3
Collabora Online : 3.3.13

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Can you provide us additional infos via logcat? https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat

**Comment by JackFish1** [comment]
Problem solved !
When I started today to produce the requested infos everything works fine.
Yesterday I made an update of the apache2-Server and modified the configuration of the header by removing
<add_header Feature-Policy "accelerometer 'none'; autoplay 'self'; geolocation 'none'; midi 'none'; sync-xhr 'self' ; microphone 'self'; camera 'self'; magnetometer 'none'; gyroscope 'none'; speaker 'self'; fullscreen 'self'; payment 'none'; usb 'none'";>
to
<IfModule mod_headers.c>
Header always set Strict-Transport-Security "max-age=15768000; preload"
Header set Referrer-Policy "strict-origin-when-cross-origin"
Header set X-Content-Type-Options "nosniff"
Header always set X-Frame-Options "SAMEORIGIN"
</IfModule>
Thank you for your quick response and best regards

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
Am Freitag, 19. Juli 2019 12:48 schrieb Tobias Kaminsky <notifications@github.com>:
